CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ Hello. 👋
 
 freeCodeCamp.org is possible, thanks to thousands of kind volunteers like you. We are grateful for your contributions and are excited to welcome you aboard.
 
-We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It's only 196 words long.
+We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It's only 197 words long.
```

Comment:
```suggestion
We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It takes only two minutes to read.
```

Reply:
So we are treating hyphenated words as two words?

Reply:
It doesn't matter.

Reply:
It think it is if we are going through the trouble of even mentioning how many words there are in the COC.

Reply:
Please see the suggested changes.